### DISCUSSION TITLE
Request for culture variable

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [PowerShellOrg/Plaster#376] [Ideas]
It would be useful to have a Plaster variable to return the current culture such as en-US. This would make it easier to add a directive to create a culture-specific folder for a module's external help.

**Comment by rkeithhill** [comment]
That should be easy enough to add.  For now, you could make that a parameter and perhaps default it to `en-US`.

**Comment by jdhitsolutions** [comment]
I knew it wouldn't be difficult to go that route. I teach in other countries and having a Plaster template that is culture aware would be useful.
